src/Evolution/Systems/GrMhd/ValenciaDivClean/Fluxes.cpp

```diff
@@ -65,9 +65,9 @@ void fluxes_impl(
       tilde_s_flux->get(i, j) = tilde_s.get(j) * transport_velocity->get(i) -
                                 lapse_b_over_w.get(j) * tilde_b.get(i);
       tilde_b_flux->get(i, j) =
-          tilde_b.get(j) * transport_velocity->get(i) +
-          get(lapse) * (get(tilde_phi) * inv_spatial_metric.get(i, j) -
-                        spatial_velocity.get(j) * tilde_b.get(i));
+          tilde_b.get(j) * transport_velocity->get(i)
+        -  transport_velocity->get(j) * tilde_b.get(i) +
+        get(lapse) * (get(tilde_phi) * inv_spatial_metric.get(i, j));
```

Comment:
Run this though clang-format.  Their documentation has instructions on how to use it through emacs or git: https://clang.llvm.org/docs/ClangFormat.html .  (Our repo has style options configured in a top-level `.clang-format` file, so you shouldn't need to configure anything yourself.)